addons/common/functions/fnc_getVehicleIcon.sqf

```diff
@@ -25,7 +25,7 @@ private _objectType = if (_object isEqualType objNull) then {
 } else {
     _object
 };
-private _cachedValue = GVAR(vehicleIconCache) getVariable _objectType;
+private _cachedValue = GVAR(vehicleIconCache) getOrDefault [_objectType];
```

Comment:
```suggestion
private _cachedValue = GVAR(vehicleIconCache) get _objectType;
```

Reply:
@veteran29 does just `get` return nil when the key is missing?

I assumed it didn't since the default value is optional in the other version.

Reply:
Wait... by definition it must. I'm out of SQF practice so was thinking there'd be an exception.